test.js

```diff
@@ -27,6 +23,10 @@ posts.forEach(element => {
 });
 console.timeEnd('for each');
 
```

Comment:
sum = 0, is missing before reduce execution.
@kapv89

Reply:
I tried after adding sum=0. Still reduce does better than for-loop. I think the difference is because of JIT. Technically, you should use large, random arrays for benchmarks.

For example:
```js
const {range} = require('lodash');
const randArr1 = range(0, 10000).map(() => Math.random());
```

Reply:
<img width="712" alt="screen shot 2018-05-10 at 3 10 21 pm" src="https://user-images.githubusercontent.com/17122343/39863448-4d2dee30-5464-11e8-84cb-eceeea765c48.png">
Chrome result and yes, you are a right JIT and large data set. But i still doubt about the for loop vs js function. I will check and update.
Anyway, thanks for the PR @kapv89  👍